app/src/main/java/com/vero/sibs/ui/registration/utils/impl/NameValidatorImpl.kt

```diff
@@ -0,0 +1,9 @@
+package com.vero.sibs.ui.registration.utils.impl
+
+import com.vero.sibs.ui.registration.utils.NameValidator
+
+class NameValidatorImpl : NameValidator {
+    override fun nameIsValid(name: String): Boolean {
+        return name.length in 3..30
+    }
+}
```

Comment:
This class implementation looks like an example of overcoding and breaking the YAGNI principle as well as the class below.

As for me defining a validator interface and a batch of implementations can be useful e.g if you're passing somewhere pairs of views and validators.